### PR TITLE
[HOSTING-492][HOSTING-512] add Kotlin 1.x Standard Library

### PR DESCRIPTION
**Pull request by casz**
# Description

- [x] https://github.com/casz/kotlin-v1-stdlib-jdk8-plugin
- [x] https://github.com/casz/kotlin-plugin-pom
- [x] https://issues.jenkins-ci.org/browse/HOSTING-492
- [x] https://issues.jenkins-ci.org/browse/HOSTING-512

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by casz** [comment]
Would you prefer a different group id? Do you want different artifact ids?

**Comment by casz** [comment]
Each plugin using this plugin pom will have the same group as we do.

**Comment by daniel-beck** [comment]
Group ID looks reasonable. This just needs the hosting request implemented; we no longer support publication of new plugins hosted outside the jenkinsci organization.

File: permissions/plugin-kotlin-v1-stdlib-jdk7.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "kotlin-v1-stdlib-jdk7"
```
Comment (oleg-nenashev) [inline permissions/plugin-kotlin-v1-stdlib-jdk7.yml]: Not sure it's actually required anymore. 2.60+ is probably enough for new plugins

Comment (casz) [reply permissions/plugin-kotlin-v1-stdlib-jdk7.yml]: Your properly right lets avoid it 👍

**Comment by casz** [comment]
The Kotlin plugin pom is not overriding the group id anymore, good or bad ❓

**Comment by casz** [comment]
@daniel-beck @oleg-nenashev we should be good to go 👍

**Pull request merged**